api/src/db/repository/user-repository.ts

```diff
@@ -10,8 +11,14 @@ export class UserRepository {
     return this.manager.findOne(User, { email });
   }
 
+  async getById(id: string) {
+    return this.manager.findOne(User, { id });
+  }
+
   async register(email: string, passwordHash: string) {
+    const id = uuid();
```

Comment:
there is database level uuid fields and typeorm support them. please switch the solution